src/components/spinner/index.tsx

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 
 import SC from './styled';
 
-const Spinner: React.FC = () => <SC.Spinner />;
+const Spinner: React.FC = () => (
```

Comment:
Endringer i spinner-komponent kan squashes med første commit.

Reply:
Da har jeg squash'a de to committene!